internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -133,6 +135,7 @@ func (s *BackendService) Template() (string, error) {
 		EnvControllerLambda:      envControllerLambda.String(),
 		Storage:                  convertStorageOpts(s.manifest.Name, s.manifest.Storage),
 		Network:                  convertNetworkConfig(s.manifest.Network),
+		DeploymentConfiguration:  deployConfig,
```

Comment:
if `getDeployConfig()` doesn't return an error, let's move the call down to here
```suggestion
		DeploymentConfiguration:  getDeployConfig(aws.StringValue(s.manifest.DeployConfig.Rolling)),
```